packages/parcel-transformer/src/index.ts

```diff
@@ -127,8 +132,7 @@ export default new Transformer<ParcelTransformerOpts>({
         config.extract && [
           '@compiled/babel-plugin-strip-runtime',
```

Comment:
Ah, I see, this plugin is used by the Babel pass _inside_ the existing Compiled transformer - that makes sense, you're not adding an extra Babel plugin to the Babel config for the project using Compiled.

So this will be addressed by whatever future endeavour we go on in order to make Compiled when used with Parcel not have an internal dependency on Babel :)